packages/devextreme-scss/scss/widgets/generic/chat/_colors.scss

```diff
@@ -39,4 +41,5 @@ $chat-messagelist-empty-prompt-color: null !default;
   $chat-messagelist-empty-icon-background-color: lighten($base-bg, 7.84) !default; // #3E3E3E
   $chat-messagelist-empty-message-color: lighten($base-text-color, 12.94) !default; // #FFFFFF
   $chat-messagelist-empty-prompt-color: darken($base-label-color, 38.04) !default; // #7D7D7D
+  $chat-messagelist-day-header-color: darken(#dedede, 37.25) !default; // #7F7F7F
```

Comment:
We shouldn't to use colors directly inside _colors.scss file. Extract this value to separate variable or to use base variable.

Reply:
https://github.com/DevExpress/DevExtreme/pull/28160/commits/7207cd514ca4374c8f263c6bc5e49c889e8e4925